app/services/idv/steps/doc_auth_base_step.rb

```diff
@@ -206,6 +208,22 @@ def log_document_error(get_results_response)
                                                get_results_response&.result_code&.name.to_s)
       end
 
+      def document_capture_session_uuid_key
+        :document_capture_session_uuid
+      end
+
+      def verify_step_document_capture_session_uuid_key
```

Comment:
What do you think about making these constants?

Reply:
I think it's a horse apiece, so I don't feel strongly either way, but am happy to change it